torchvision/datasets/imagenette.py

```diff
@@ -0,0 +1,104 @@
+from pathlib import Path
+from typing import Any, Callable, Optional, Tuple
+
+from PIL import Image
+
+from .folder import find_classes, make_dataset
+from .utils import download_and_extract_archive, verify_str_arg
+from .vision import VisionDataset
+
+
+class Imagenette(VisionDataset):
+    """`Imagenette <https://github.com/fastai/imagenette#imagenette-1`_ image classification dataset.
+
+    Args:
+        root (string): Root directory of the Imagenette dataset.
+        split (string, optional): The dataset split. Supports ``"train"`` (default), and ``"val"``.
+        size (string, optional): The image size. Supports ``"full"`` (default), ``"320px"``, and ``"160px"``.
```

Comment:
Maybe it's more convenient if we don't force the "px" suffix? I don't think there's any ambiguity here. (These could also be `int`). No strong opinion.

```suggestion
        size (string, optional): The image size. Supports ``"full"`` (default), ``"320"``, and ``"160"``.
```